extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/theta/sql/ThetaSketchSqlAggregatorTest.java

```diff
@@ -423,6 +477,9 @@ public void testAvgDailyCountDistinctThetaSketch() throws Exception
   @Test
   public void testThetaSketchPostAggs() throws Exception
   {
+    // Can't vectorize due to CONCAT expression.
+    cannotVectorize();
```

Comment:
hmm, after #11010 `CONCAT` should be vectorizable, did this fail to vectorize? (You might need to add context to vectorize expressions, I can't remember off the top of my head for tests but it isn't enabled by default yet...)

Reply:
Adding the context parameter enabled this test to pass — thanks.